templates/services/merge/validator/prysm.tmpl

```diff
@@ -1,39 +1,10 @@
 {{/* prysm.tmpl */}}
 {{ define "validator" }}
-  validator-import:
-    image: ${VL_IMAGE_VERSION}{{with .VlCustomCfg}}
-    depends_on:
-      config_consensus:
-        condition: service_completed_successfully{{end}}
-    networks:
-      - sedge
-    container_name: validator-import-client
-    volumes:
-      - ${VL_DATA_DIR}:/data
-      - ${KEYSTORE_DIR}:/keystore{{if .VlCustomCfg}}{{if .VlRemoteCfg}}
-      - ./config.yml:/network_config/config.yml{{end}}{{end}}
-    command: |
-      accounts import{{if .VlCustomCfg}}{{if .VlRemoteCfg}}
-      --chain-config-file=/network_config/config.yml{{end}}{{else}}
```

Comment:
Same for this flag in Prysm's cmd

Reply:
Please check this commit https://github.com/NethermindEth/sedge/pull/208/commits/09743fd0d382943870ce2066823bbcdded15964a